### DISCUSSION TITLE
New functions for Power Platform Tool

### DISCUSSION THREAD

**Discussion by kangjin62** [microsoft/powerplatform-build-tools#919] [Ideas]
Hello,

Could it be possible for Power Platform Tool to implement new tasks for ADO where the tool is able to do the following tasks:
1. Create a **patch** solution
2. Retrieve latest **patch** solution
3. Retrieve a solution (via solution name)

Currently my team is dependent on the 3 tasks listed above and we are utilizing a different tooling to complete the tasks. However with attempting to use FIC/Managed Identities, we cannot use the tooling we have been utilizing as that tooling uses client secret and connection string for authentication and thus we are looking for alternative solutions. 

Thank you!

**Comment by rajyraman** [comment]
@kangjin62 - Please see https://learn.microsoft.com/en-us/power-platform/alm/update-solutions-alm#create-solution-patches

![image](https://github.com/microsoft/powerplatform-build-tools/assets/5035266/e96e49fb-eef0-4663-b9be-06cf7891f860)
